### PR TITLE
Issue 1050: Resolve health check error

### LINKED ISSUE #1050: /health check marking sandbox as unhealthy during EPP calls
### Current Behavior

When clicking the 'manage' icon next to an approved domain, or when adding nameservers/security email, at seemingly random times the sandbox is producing a 500 error due to the sandbox itself restarting. The logs show that the /health check timeout after 1 second which overlaps with while it was sending, receiving and processing EPP messages.

### Expected Behavior

The slow down from using EPP should not result in a 500 or a sandbox restart

### Steps to Reproduce

1.click the manage icon on an approved domain
2. see 500 error

*may have to repeat this many times in a row to get it to throw an error as it doesn't happen each time.

### Environment

_No response_

### Additional Context

_No response_

### Issue Links

_No response_

### PR DESCRIPTION
**Pull request by zandercymatics**
## Ticket

Resolves #1050 

## Changes

- Increased timeout for each .yaml file from 1 second -> 40 seconds

## Context for reviewers
Our EPPLib calls take a considerable amount of time, causing our sandboxes to timeout and crash. As a temporary measure, we can increase the timeout period to resolve this.

#### Satisfied acceptance criteria and met development standards

- [x] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

**Review by abroddrick — CHANGES_REQUESTED** [review]
See previous comment

File: ops/scripts/manifest-sandbox-template.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ applications:
   command: ./run.sh
   health-check-type: http
   health-check-http-endpoint: /health
+  health-check-invocation-timeout: 30
```
[the remark this answers is not in the document]
Comment (zandercymatics) [reply ops/scripts/manifest-sandbox-template.yaml:14]: @abroddrick Ah! Yes that is precisely why then. Makes total sense why the command worked but making the change didn't, not sure how I did not catch that! I'll remove the shell change and all this to every .yaml, seems reasonable

**Comment by zandercymatics** [comment]
@abroddrick Okay I got those changes in. I don't have the means to test this on other peoples sandboxes, but this seems to be working okay when I tested it

**Comment by abroddrick** [comment]
@zandercymatics was waiting on the checks to not fail, just clicked re-run on the deploy. If it passes I can approve

**Comment by abroddrick** [comment]
Okay the re-run of your checks worked. Fyi it can be good to check to make sure things aren't failing on the PR if it's sitting up for review

**Review by abroddrick — APPROVED** [review]
LGTM

**Pull request merged**